Sources/Gravatar/Network/Services/AvatarService.swift

```diff
@@ -90,15 +90,22 @@ public struct AvatarService: Sendable {
         }
     }
 
-    package func update(_ altText: String?, rating: AvatarRating?, avatarID: String, accessToken: String) async throws {
+    @discardableResult
+    package func update(
+        _ altText: String?,
```

Comment:
```suggestion
        altText: String?,
```

Now that we have multiple parameters that we can "update", I think the calling site might read better if we name the first parameter.  What do you think?

Reply:
Let's go with this change 👍